Wikipedia/UI-V5/WMFArticleContainerViewController.m

```diff
@@ -262,29 +337,34 @@ - (void)viewWillTransitionToSize:(CGSize)size withTransitionCoordinator:(id<UIVi
     [super viewWillTransitionToSize:size withTransitionCoordinator:coordinator];
     [coordinator animateAlongsideTransition:^(id < UIViewControllerTransitionCoordinatorContext > context) {
         [self updateInsetsForArticleViewController];
-        [self.previewController updatePreviewWithSizeChange:size];
     } completion:NULL];
 }
 
 - (void)updateInsetsForArticleViewController {
-    CGFloat topInset = [self.navigationController.navigationBar frame].size.height
-                       + [[UIApplication sharedApplication] statusBarFrame].size.height;
-
-    UIEdgeInsets adjustedInsets = UIEdgeInsetsMake(topInset,
-                                                   0.0,
-                                                   self.tabBarController.tabBar.frame.size.height,
-                                                   0.0);
+    // TODO: remove
```

Comment:
Was planning to do all the articleVC deletions in one pass to keep the diff manageable, but can remove this part now if you prefer.